x/relationships/client/rest/tx.go

```diff
@@ -0,0 +1,91 @@
+package rest
+
+import (
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/desmos-labs/desmos/x/relationships/types"
+	"github.com/gorilla/mux"
+)
+
+func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router) {
+	r.HandleFunc("/relationships/create/{address}", createRelationshipHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/relationships/delete/{address}", deleteRelationshipHandler(cliCtx)).Methods("DELETE")
```

Comment:
I think we can delete the `/create/` and `/delete/` parts of the query, and maybe the `/{address}` as well, requiring the user to put it into the body of the request instead. What do you think?

```suggestion
	r.HandleFunc("/relationships", createRelationshipHandler(cliCtx)).Methods("POST")
	r.HandleFunc("/relationships", deleteRelationshipHandler(cliCtx)).Methods("DELETE")
```

Reply:
I think there will not be much difference. The only concern I have is if it's possibile to have two endpoints with the exact same name

Reply:
@bragaz Yes, it is cause the REST method will be different. In the first one it will be `POST`, and the other one it will be `DELETE`. Removing the `/create` and `/delete` parts will make the endpoint follow the REST guidelines (no verbs inside endpoint names, only nouns). And removing the `{address}` variable will make it coherent with other requests as well